transport/grpc/config.go

```diff
@@ -69,7 +72,6 @@ type TransportConfig struct {
 	ServerMaxSendMsgSize int                 `config:"serverMaxSendMsgSize"`
 	ClientMaxRecvMsgSize int                 `config:"clientMaxRecvMsgSize"`
 	ClientMaxSendMsgSize int                 `config:"clientMaxSendMsgSize"`
-	ClientTLS            bool                `config:"clientTLS"`
```

Comment:
Was this ever released? If so, removing this is a breaking change.

Reply:
It was never released. I left it on dev, but trimmed it out for the last release.